getting_started/install_linux.md

```diff
@@ -45,58 +63,80 @@ If Docker is not present, you can install Docker On Fedora/RHEL systems using [i
 
 +++ Update your OS
 To ensure a smooth install, we recommend updating your OS:
+- For Fedora:
 ```shell
-# For Fedora:
-$ sudo dnf update -y
-
-# For Ubuntu Focal:
-$ sudo apt update -y
-
-# For Amazon Linux 2:
-$ sudo yum update -y
+sudo dnf update -y
+```
+- For Ubuntu Focal:
+```shell
+sudo apt update -y
+```
+- For Amazon Linux 2:
+```shell
+sudo yum update -y
 ```
 
 +++
 
 ## Install Kontain
 ```shell
-# create the kontain folder for install
-$ sudo mkdir -p /opt/kontain ; sudo chown root /opt/kontain
+sudo mkdir -p /opt/kontain ; sudo chown root /opt/kontain
+curl -s https://raw.githubusercontent.com/kontainapp/km/current/km-releases/kontain-install.sh | sudo bash
+```
 
-# download and install the latest Kontain release
-$ curl -s https://raw.githubusercontent.com/kontainapp/km/current/km-releases/kontain-install.sh | sudo bash
-...
+This installed Kontain in /opt/kontain directory and configured the Kontain runtime for docker (/etc/docker/daemon.json) and podman (~/.config/containers/containers.conf). 
 
-# This installed Kontain in /opt/kontain and added the Kontain runtime (/opt/bin/km) to docker in /etc/docker/daemon.json. 
-# This also executed a smoke test of the unikernel virtual machine.
+This also executed a smoke test of the unikernel virtual machine.
 
-# check for Kontain in /opt/kontain folder
-$ ls -l /opt/kontain
-...
-drwxr-xr-x 1 smijar  121 296 Nov 10 09:32 bin
-drwxr-xr-x 1 smijar  121 140 Nov 10 09:32 lib
-drwxr-xr-x 1 smijar  121 194 Nov 10 09:32 runtime
-...
+## Verify Kontain Install
+1. Check for Kontain in /opt/kontain folder
+```shell
+ls -l /opt/kontain
+drwxr-xr-x   4 smijar  121   4096 Sep 16 18:03 alpine-lib
+drwxr-xr-x   2 smijar  121   4096 Oct 10 16:20 bin
+drwxr-xr-x   3 smijar  root   4096 Oct  5 17:10 config
+drwxr-xr-x   7 smijar  121   4096 Sep 16 17:24 examples
+drwxr-xr-x   2 smijar  121   4096 Sep 16 18:03 include
+-rwxr-xr-x   1 smijar  121 370035 Sep 16 18:03 kkm.run
+drwxr-xr-x   2 smijar  121   4096 Sep 16 18:03 lib
+drwxr-xr-x   2 smijar  121   4096 Sep 16 18:03 runtime
+drwxr-xr-x   2 smijar  root   4096 Oct  7 09:25 tests
 ```
 
-## Verify Kontain install
+2. Verify Kontain install
+- check if Kontain monitor can run the unikernel
 ```shell
-# check if Kontain monitor can run the unikernel
-$ /opt/kontain/bin/km /opt/kontain/tests/hello_test.km
-
-# check if docker can run with kontain as runtime
-$ docker run --runtime=krun kontainguide/hello-kontain
+/opt/kontain/bin/km /opt/kontain/tests/hello_test.km
+```
+Output will look like 
+:::custom-shell-output
 ```
+Hello, world
+Hello, argv[0] = '/opt/kontain/tests/hello_test.km'
+```
+:::
 
+- check if docker can run with kontain as runtime
+```shell
+docker run --runtime=krun kontainguide/hello-kontain
+```
+Output will look like 
+:::custom-shell-output
```

Comment:
This line needs to be fixed I think.  Was meant to be some documentation directive I think

Reply:
It is documentation directive - custom block